server/src/main/java/io/seata/server/coordinator/DefaultCore.java

```diff
@@ -300,11 +290,7 @@ public GlobalStatus rollback(String xid) throws TransactionException {
     public boolean doGlobalRollback(GlobalSession globalSession, boolean retrying) throws TransactionException {
         boolean success = true;
         // start rollback event
-        eventBus.post(new GlobalTransactionEvent(globalSession.getTransactionId(),
-                GlobalTransactionEvent.ROLE_TC, globalSession.getTransactionName(),
-                globalSession.getApplicationId(),
-                globalSession.getTransactionServiceGroup(), globalSession.getBeginTime(),
-                null, globalSession.getStatus()));
+        SessionHelper.postTcSessionBeginEvent(globalSession);
```

Comment:
`postTcSessionBeginEvent`  is not appropriate here.

Reply:
will optimize later